_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -204,7 +204,18 @@ FIXME:harding to update Tuesday
   users of time-sensitive transactions (such as in LN and many other
   contract protocols).
 
-- [Bitcoin Core #19077][] wallet: Add sqlite as an alternative wallet database and use it for new descriptor wallets FIXME:dongcarl
+- [Bitcoin Core #19077][] adds a new SQLite database backend for wallet records.
+  Unlike the existing Berkeley DB database backend, SQLite is designed and
+  tested to be resilient against crash and power loss corruption. SQLite 3 also
+  has a backward compatibility guarantee, the lack of which in Berkeley DB is
+  the reason why Bitcoin Core has to target Berkeley DB version 4.8
+  specifically. This backward compatibility guarantee also means that Bitcoin
+  Core can benefit from the bug fixes and system compatibility improvements in
+  future releases of SQLite 3. In the upcoming v0.21 release, legacy wallets
```

Comment:
Not sure that "This backward compatibility guarantee also means that Bitcoin Core can benefit from the bug fixes and system compatibility improvements in future releases of SQLite 3" is entirely relevant to readers. Reviewers: feel free to remove!

Reply:
I agree that this is getting into the weeds a bit. Let's cut it.

Reply:
I'd also suggest removing the backwards-compatibility sentence before. bitcoind can be built with newer versions of bdb, but by default we don't (for reasons that don't entirely make sense to me).